mcs/class/System/System.Net.NetworkInformation/Win32NetworkInterfaceMarshal.cs

```diff
@@ -45,7 +48,34 @@ class Win32NetworkInterface {
 					GetNetworkParams (ptr, ref len);
 					ptr = Marshal.AllocHGlobal(len);
 					GetNetworkParams (ptr, ref len);
-					fixedInfo = Marshal.PtrToStructure<Win32_FIXED_INFO> (ptr);
+					var fixedInfoUnmarshal = Marshal.PtrToStructure<Win32_FIXED_INFO_Marshal> (ptr);
+
+					unsafe string GetStringFromMultiByte(byte * bytes)
+					{
+						var len1 = MultiByteToWideChar(0, 0, bytes, -1 , null, 0);
```

Comment:
The first parameter is code page. I couldn't determine from reading the docs what `0` represents in this case.